openresty-build-tools/kong-ngx-build

```diff
@@ -232,6 +232,7 @@ main() {
           notice "Downloaded: $(sha256sum "openssl-$OPENSSL_VER.tar.gz")"
         fi
         tar -xzf openssl-$OPENSSL_VER.tar.gz
+        ln -s openssl-${OPENSSL_VER} openssl
```

Comment:
Nice! Looks like a good way to let the later stages of the build stop caring about `$DEPENDENCY_VERSION`.

Is there a reason to symlink instead of rename? I don't really have an underlying preference--mostly just curious.

Reply:
it gives verbosity and convenience:
verbosity `ls` will tell you the specific openssl version convenience `cd openssl` works.

We also require the versioned directory for backwards compatibility reasons :(